traits.yml

```diff
@@ -45,6 +55,57 @@ traits:
 
               This must be a valid URL, so special characters need to
               be encoded accordingly.
+          mimeType:
+            type: string
+            description: >
+              The MIME type of the data referenced by the location
+              property.
+
+
+              See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+          isTemplated:
+            type: boolean
+            description: >
+              The URL in the location property contains variables (post
+              decoding) that must be expanded before any loading is
+              attempted.
+
+
+              Variables use the fmt library syntax (eg: "image.{frame:04d}.{view}.exr").
+              See https://hackingcpp.com/cpp/libs/fmt.html for more
+              details.
+
+
+              Not all variables are available in all hosts, but the
+              following common variables are defined:
```

Comment:
It was supposed to mean that the MediaCreation library defines them and their meaning, but not all hosts support all of the.

Reply:
Hows https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/71/commits/7bc55f67827d7a377944c65b72cfef6e732cad22

Reply:
Yeah, that's better. I still think the usage of the word `defined` is a bit troublesome, as it would imply to me some sort of actual `#DEFINE` or environment variable that the mediacreation project itself is doing. However, saying "defined in the ecosystem" works a little to alleviate that.